pslogin/src/main/scala/services/vehicle/VehicleService.scala

```diff
@@ -159,13 +149,11 @@ class VehicleService extends Actor {
       VehicleEvents.publish(
         VehicleServiceResponse(s"/${zone.Id}/Vehicle", Service.defaultPlayerGUID, VehicleResponse.LoadVehicle(vehicle, vtype, vguid, vdata))
       )
-      vehicleDelayedDecon ! DelayedDeconstructionActor.UnscheduleDeconstruction(vguid)
-      vehicleDelayedDecon ! DelayedDeconstructionActor.ScheduleDeconstruction(vehicle, zone, 600L) //10min
+      vehicleDecon forward RemoverActor.AddTask(vehicle, zone, Some(30 seconds))
```

Comment:
Probably worth adding a comment here to specify why the vehicle will be deconstructed (Stop pad being blocked) and when deconstruction will be cancelled (On player seated in driver seat successful?)